### DISCUSSION TITLE
Any plans for mac support?

### DISCUSSION THREAD

**Discussion by MattTuckman** [stirante/lol-client-java-api#23] [General]
Been enjoying using this library but would prefer to be able to develop on mac and test the app there.

 Additionally would be nice to support people who would want to play league on mac as rare as they may be 😄

**Comment by stirante** [comment]
Support for mac is added in theory, but I haven't had a chance to actually check whether it's working. The only platform specific thing is getting process info on LoL client and it's done by [this class](https://github.com/stirante/lol-client-java-api/blob/master/src/main/java/com/stirante/lolclient/ProcessWatcher.java). Specific for a mac behavior is coded [here](https://github.com/stirante/lol-client-java-api/blob/master/src/main/java/com/stirante/lolclient/PSProcessWatcher.java).

Current release might not have it yet, so you can try using latest snapshot from nexus.stirante.com (README describes how to get it). If it's not working, let me know.

**Reply by MattTuckman** [reply]
Just saw your reply. I'll test this out with the snapshot next time I'm free and let you know!

**Comment by szabolcs-dekany** [comment]
A new pr was just merged that adds macOS specific process watcher so mac support should be better now. No release for this one yet, but if you build the project locally, you should be able to test it.

**Reply by stirante** [reply]
Actually my private jenkins built it already. You can find the instructions to use it here: https://github.com/stirante/lol-client-java-api#snapshots
